mvlearn/datasets/gaussian_mixture.py

```diff
@@ -139,6 +181,12 @@ def __init__(self, mu, sigma, n, class_probs=None):
                 ]
             )
 
+        if self.shuffle:
+            np.random.seed(self.shuffle_random_state)
```

Comment:
Correct me but I believe this would shuffle y and latent independently if shuffle_random_state is None.

Reply:
Shuffle a list of indices and apply to both

Reply:
I also set the random seed before I shuffle the labels below though. I plotted both and I believe they are shuffled correctly.